test/region-testing/pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We probably need to exclude this module from mvn release command and add it to the excluded import list.

Reply:
Done